package.json

```diff
@@ -51,7 +54,9 @@
   },
   "scripts": {
     "test": "mocha",
-    "compile": "babel src --out-dir dist",
+    "compile:cjs": "babel src --out-dir dist",
+    "compile:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "compile": "npm-run-all --parallel compile:cjs compile:es",
```

Comment:
I don't really mind the extra dependency (as long as it isn't huge) since its dev-only anyways . What do you mean exactly with "run its arguments with yarn"?